ilc/client.js

```diff
@@ -144,11 +145,36 @@ function getCurrentBasePath() {
     return currentPath.basePath;
 }
 
+
+let prevUrl;
+let currentUrl = window.location.pathname + window.location.search;
 window.addEventListener('single-spa:before-routing-event', () => {
+    // !!! TODO remove it after fix https://github.com/single-spa/single-spa/issues/484
+    // and add "base.remove();" after first time of using
+    prevUrl = currentUrl;
+    currentUrl = window.location.pathname + window.location.search;
+    if (currentUrl !== prevUrl) {
+        const base = document.querySelector('base');
```

Comment:
Search in head + I would prefer `getElementsByTagName` here

Reply:
I would prefer getElementsByTagName here
1) getElementsByTagName return array of all items and we will take the first one. querySelector is faster because it looks only for the first one.
2) querySelector return static array of NodeList unlike getElementsByTagName returns dynamic HTMLCollection.

Search in head 
1) search engines put head tags of the page to body. so it's better to search in head and body.